### PR TITLE
Feature comment pahts

### PR DESCRIPTION
**Pull request by kalomak**

### CODE REVIEW

**Review by joonalillfors — DISMISSED** [review]
sihp it

File: client/src/components/route-container/route-container.js [hunk 1]
```diff
@@ -48,24 +48,28 @@ class RouteContainer extends React.Component {
...
+              {route[1]} {this.pluralize('comment', route[1])} at{' '}
+              <a href={route[0]}>{route[0]}</a>
+            </p>
+          ))}
+          <button type="button" onClick={this.hide}>
+            Collapse
+          </button>
+        </div>
+      )
+    } else {
```
Comment (HenrikHelenius) [inline client/src/components/route-container/route-container.js]: no curlies

File: client/src/components/route-container/route-container.js [hunk 2]
```diff
@@ -48,24 +48,28 @@ class RouteContainer extends React.Component {
...
-          hide
-        </button>
-      </div>
-    )
+    if (amountsByRoute.length > 0) {
+      return (
+        <div
+          className={css('route-container', { hidden: this.state.hidden })}
+          onClick={this.expand}
+          // TODO: element can be focused when hidden = true
```
Comment (HenrikHelenius) [inline client/src/components/route-container/route-container.js]: wtf

**Review by HenrikHelenius — DISMISSED** [review]

File: client/src/components/route-container/route-container.js (hunk 2 above)
Comment (kalomak) [reply client/src/components/route-container/route-container.js]: done

**Review by shahruzmannan — APPROVED** [review]

**Pull request merged**